taletime/lib/main.dart

```diff
@@ -81,9 +86,23 @@ class Providers extends StatelessWidget {
   Widget build(BuildContext context) {
     final languageProvider = Provider.of<LocaleProvider>(context);
     final themeProvider = Provider.of<ThemeProvider>(context);
-    return MaterialApp(
+
+    return MaterialApp.router(
+      routerConfig: GoRouter(
+        routes: [
+          GoRoute(
+            path: "/",
+            builder: (_, __) => HomePage(),
```

Comment:
Can't we just remove the _ and just have ()=>?

Reply:
Nope, does not work.